MovieApp/Screens/Search/SearchViewController.swift

```diff
@@ -7,10 +7,157 @@
 
 import UIKit
 
+private enum Constants {
+	static let searchCLVC = "SearchCollectionViewCell"
+	static let searchCollectionReusableView = "SearchCollectionReusableView"
+	static let searchDetailCell = "SearchDetailCell"
+	static let detailVC = "DetailsViewController"
+}
+
+protocol ISearchViewController: AnyObject {
+	func showMovies()
+	func showError(_ errorMessage: String)
+}
+
 class SearchViewController: BaseViewController {
+	// MARK: - IBOutlets
+	@IBOutlet weak var searchCollectionView: UICollectionView!
```

Comment:
```suggestion
	@IBOutlet weak private var searchCollectionView: UICollectionView!
```
@dong2810 : Make it private